src/Command/CronDisable.php

```diff
@@ -0,0 +1,72 @@
+<?php
+/**
+ * Copyright © Magento, Inc. All rights reserved.
+ * See COPYING.txt for license details.
+ */
+declare(strict_types=1);
+
+namespace Magento\MagentoCloud\Command;
+
+use Magento\MagentoCloud\App\GenericException;
+use Magento\MagentoCloud\Step\Deploy\DisableCron;
```

Comment:
Let's not use steps in command because it's different domain models. Create class which manages cron and use it explicitly.

Reply:
Done.